conans/cli/commands/search.py

```diff
@@ -20,8 +20,19 @@ def search(conan_api, parser, *args, **kwargs):
     args = parser.parse_args(*args)
 
     remotes = conan_api.get_active_remotes(args.remote)
-
     results = []
+
     for remote in remotes:
-        results.append(conan_api.search_remote_recipes(args.query, remote))
+        error = None
```

Comment:
This implementation is not fully clear to me. If this command is going to be an alias of ``conan list recipes --all``, then maybe it should be implemented as such? I would expect to see that equivalence directly in code.

Reply:
Completely agree but I think we should open another PR to change it.